src/screens/Podcast/components/EpisodeItem/common/constants/index.ts

```diff
@@ -0,0 +1,3 @@
+const INCREASE_VALUE_FOR_LIST = 1
```

Comment:
We could name it simpler. For example, `ARRAY_OFFSET`. And let's put it to `/src/common/constants/array`